packages/components/organisms/f-header/src/components/Logo.vue

```diff
@@ -46,33 +48,33 @@ export default {
         headerBackgroundTheme: {
             type: String,
             default: 'white'
+        },
+        isOpen: {
+            type: Boolean,
+            default: false
         }
     },
     computed: {
         iconComponent () {
             return `${this.theme}-logo`;
         },
         iconClassName () {
-            return `c-icon--${this.theme}`;
+            return this.$style[`c-icon--${this.theme}`];
         },
         linkAltText () {
             return `Go to ${this.companyName} homepage`;
         },
-        logoColourModifier () {
-            switch (this.headerBackgroundTheme) {
-                case 'transparent':
-                    return 'c-icon--onTransparentBg';
-                case 'highlight':
-                    return 'c-icon--onHighlightBg';
-                default:
-                    return '';
-            }
+        isAltLogo () {
+            const isHighlight = this.headerBackgroundTheme === 'highlight';
+            const isTransparent = this.headerBackgroundTheme === 'transparent' && !this.isOpen;
+            return isHighlight || isTransparent;
```

Comment:
I would probably do:
```
...
const isTransparent = this.headerBackgroundTheme === 'transparent';
return isHighlight || (isTransparent && !this.isOpen);
```
A very minor thing in this case, but if `isHighlight` is `true` we're still calculating `isTransparent`, even though ideally we'd want to skip it.

Reply:
Ooh good point, I'll update it in the part 2. Thanks!